src/components/Token/ButtonsWrapper/index.tsx

```diff
@@ -0,0 +1,34 @@
+import LinkButton from '@/components/common/LinkButton'
+import { Button } from '@mui/material'
+import SafeLink from '@/components/common/SafeLink'
+import { type ButtonEntry } from '@/config/types'
+import css from './styles.module.css'
+
+type ButtonsWrapperProps = { buttons: ButtonEntry[] }
+
+const ButtonsWrapper = ({ buttons }: ButtonsWrapperProps) => {
+  if (!buttons || buttons.length === 0) return null
```

Comment:
The type defines `buttons` as defined.

```suggestion
  if (buttons.length === 0) return null
```